t/node/consumer-group.t

```diff
@@ -310,3 +310,140 @@ world
     }
 --- response_body
 bar
+
+
+
+=== TEST 5: data encryption
+--- yaml_config
+apisix:
+    data_encryption:
+        enable: true
+        keyring:
+            - edd1c9f0985e76a2
+--- config
+    location /t {
+        content_by_lua_block {
+            local json = require("toolkit.json")
+            local t = require("lib.test_admin").test
+            local etcd = require("apisix.core.etcd")
+            local code, body = t('/apisix/admin/consumer_groups/company_a',
```

Comment:
Why use consumer_groups?

Reply:
make sure that data encryption works well with consumer groups